src/AppBundle/Resources/views/Person/show.json.twig

```diff
@@ -0,0 +1,62 @@
+
+   {
+        
+   "@context": "http://schema.org",
+   "@type": "Person",
+
+   "url": "{{ url('person_show', {'id': person.id}) }}",
+   "familyName": "{{ person.firstName }}",
+   "givenName": "{{ person.lastName }}",
+   {% if person.title %}
+   "jobTitle": "{{ person.title }}",
+   {% endif %}
+   "birthDate": "{{ person.dob }}",
```

Comment:
birthDate and deathDate can be null. Will that be OK with the json parsers and validators?